### PR TITLE
add PrivateHostedZoneID and PrivateHostedZoneRoleArn in Cluster type

### PR DESCRIPTION
**Pull request by marcolan018**
Signed-off-by: marcolan018 <llan@redhat.com>

### CODE REVIEW

File: model/clusters_mgmt/v1/cluster_type.model [hunk 1]
```diff
@@ -225,5 +225,11 @@ class Cluster {
 	DeleteProtection DeleteProtection
 
 	// Details for `htpasswd` identity provider.
-	Htpasswd HTPasswdIdentityProvider	
+	Htpasswd HTPasswdIdentityProvider
+
+	// ID of private hosted zone
```
Comment (gdbranco) [inline model/clusters_mgmt/v1/cluster_type.model:230]: It needs the ending '.' to include it in the openapi (swagger) page

**Review by ciaranRoche — APPROVED** [review]
/lgtm

File: model/clusters_mgmt/v1/cluster_type.model (hunk 1 above)
Comment (ciaranRoche) [reply model/clusters_mgmt/v1/cluster_type.model:230]: TIL

Comment (marcolan018) [reply model/clusters_mgmt/v1/cluster_type.model]: @gdbranco 
thanks for letting me know this
I already added the ending ".". Please help to review again

**Review by gdbranco — APPROVED** [review]

**Pull request merged**